lib/styles/settings/_buttons.scss

```diff
@@ -1,4 +1,5 @@
 @import 'colors';
```

Comment:
old colors are still imported

Reply:
Fixed. @kvas-damian temporary i need to comment styles in `IconButton` when I removed old structures/colors from button in `IconButton` were errors. Whole `IconButton` will be replaced in the next PR